.github/ISSUE_TEMPLATE/falsepositive.yml

```diff
@@ -0,0 +1,112 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+name: False Positive Report
+description: File a Phishing report
+labels: ["False Positive"]
+assignees:
+  - mitchellkrogza
+  - funilrys
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We understand being listed on a Phishing Database like this can be frustrating and embarrassing for many web site owners.
+
+        The first step is to remain calm.
+        
+        The second step is to rest assured one of our maintainers will address your issue as soon as possible.
+
+        Please make sure you have provided as much information details as possible to help speed up the process.
+
+  - type: textarea
+    id: record
+    attributes:
+      label: Domain/URL/IP(s) you believes NOT to be Phishing
+      description: |
+        Please list any domains and links listed here which you believe are a false positive.
+        
+        Please warn with "NSFW" where applicable.
+      placeholder: |
+        example.com
+        https://example.org/phishing
+        192.0.2.0/24
+      render: css
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: source
+    attributes:
+      label: More Information
+      description: |
+        How did you discover your IP-address or domain was listed in the Phishing or Phishing.Database?
```

Comment:
May be better:

```
How did you discover your IP address or domain was listed in our Phishing Database?
```

Reply:
Or replace _IP address or domain_ with _website_

Reply:
Even better @iam-py-test !

```
How did you discover your website was listed in our Phishing Database?
```

cc @spirillen

Reply:
To shoot that one to the corner... (commenting)
I had written website to begin with, but, how many mail servers are not interferred because of this project?? I don't know, however I suspect this project reach far beyond website alone and thats the reason I chose domain *and* domain is more equeal what you find in the domain list... the websites are in the URL lists :wink:

Still of for a change? we have democracy. if you two vote against me I acccept. @mitchellkrogza have the veto.